### ISSUE TITLE
Audit App Headings in Claims (Benefits)

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7345]
Audit the section of the app to ensure that the accessibilityRole=(‘header’) is in place on anything that should be a heading.

- [ ] Assign a category to each "heading"
- [ ] List location for each "heading"
- [ ] List content included in each "heading"
- [ ] Listen to each "heading" in VoiceOver or check code to determine if it is currently labeled as a heading
- [ ] Determine if changes are needed (if so, include information of what the change should be)
